hwcomposer/hwcomposer_backend_v11.cpp

```diff
@@ -393,7 +687,7 @@ HwComposerBackend_v11::sleepDisplay(bool sleep)
     }
 }
 
-int HwComposerBackend_v11::getSingleAttribute(uint32_t attribute)
+int HwComposerBackend_v11::getSingleAttribute(uint32_t attribute, int dpy)
```

Comment:
When I see function like "getSingleAttribute(uint32_t attribute, int dpy)" it wass not automatically obvious to me that dpy refered to display's index number. In Qt style abbreviations are avoided (expect in fully local, obvious cases like for (int i; i < count; i++)... Could align with blankDisplay(int display, ..)

Reply:
Also would be good to consistently have display index as the first or second attribute, now both are used